Automattic/wp-calypso
# Gutenboarding - Add editor loading placeholder

#### Changes proposed in this Pull Request

Add a loading placeholder for the editor in the Gutenboarding flow as per #39246

![create-site-loading-editor](https://user-images.githubusercontent.com/28742426/73893447-ae28aa80-4847-11ea-956e-b5ba9144f0b9.gif)

This is accomplished by having a `create-site` route at the end of the Gutenboard Steps.  The idea is to update this Route to contain the create-site logic in a follow up (as well as make this redirect to another step if somehow loaded when site creation does not make sense). But I think this is in a pretty good place to test and merge, with room for some followups after #39203 and #39050 land.

#### Testing instructions



*  Run this PR, navigate through Gutenboarding, click "Create Site".
*  Verify there is a loading placeholder until the editor starts loading.
*  After creating the site, visiting `/gutenboarding/create-site` should redirect to `/gutenboarding`

Fixes #39246
